test/controllers/Phase1PassMarkSettingsControllerSpec.scala

```diff
@@ -39,7 +39,8 @@ class Phase1PassMarkSettingsControllerSpec extends UnitWithAppSpec {
     "Return a settings objects with schemes but no thresholds if there are no settings saved" in new TestFixture {
```

Comment:
Does the test corresponds to the description? Where do we assert that we get "a settings objects with schemes but no thresholds"?